plugins/inputs/elasticsearch_query/aggregation_parser.go

```diff
@@ -0,0 +1,154 @@
+package elasticsearch_query
+
+import (
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	elastic "gopkg.in/olivere/elastic.v5"
```

Comment:
As per our discussion, please prepare for later versions by
```suggestion
	elastic5 "gopkg.in/olivere/elastic.v5"
```
or similar.